packages/nextjs/src/server/protect.ts

```diff
@@ -8,7 +8,7 @@ import type {
 import { constants as nextConstants } from '../constants';
 import { SIGN_IN_URL } from './constants';
 
-type AuthProtectOptions = { redirectUrl?: string };
+type AuthProtectOptions = { unauthorizedUrl?: string; unauthenticatedUrl?: string };
```

Comment:
I went with `unauthenticatedUrl` instead of `signInUrl`. I'm not opinionated about this we can go with the initial naming.

Reply:
@nikosdouvlis now that i'm thinking about it, we may still need to do `signInUrl` because of the env variables right ?